static/styles/subscriptions.css

```diff
@@ -994,6 +994,22 @@ div.settings-radio-input-parent {
     select.stream_post_policy_setting {
         margin-bottom: 10px;
     }
+
+    .dropdown-list-widget {
+        .dropdown-toggle {
+            margin-bottom: 10px;
+            width: 325px;
+        }
+
+        ul.dropdown-menu {
+            width: 325px;
+
+            input {
+                /* 325 - 2* (9px (margin) + 1px (border) + 6px (padding)) */
+                width: 293px;
```

Comment:
This comment is super helpful, but generally we should be trying to avoid needing to do this sort of math; there's probably a way to just make the `input` fill the available space, so that it just works if any of those numbers change? Idunno, probably best as a #frontend thread, and not a blocker for this PR.